android/src/main/java/io/sentry/react/RNSentryModuleImpl.java

```diff
@@ -102,6 +103,17 @@ Activity getCurrentActivity() {
 
     public void initNativeSdk(final ReadableMap rnOptions, Promise promise) {
         SentryAndroid.init(this.getReactApplicationContext(), options -> {
+            if (rnOptions.hasKey("_metadata")) {
```

Comment:
Shouldn't this come from `event.sdk`? why `_metadata`?

Reply:
We are setting this during SDK init and we cant access `event`. 

`_metadata` it's the name of this property we inherit from javascript.

https://github.com/getsentry/sentry-javascript/blob/2aa4e94b036675245290596884959e06dcced044/packages/types/src/options.ts#L184

Reply:
Gotcha, I am just trying to understand how this ends in the `event.sdk` on RN? otherwise this info isn't even set.

Reply:
Got it, from `options._metadata.sdk` it's added to `event.sdk` in the `baseclient` implementation that RN client extends.

https://github.com/getsentry/sentry-javascript/blob/2aa4e94b036675245290596884959e06dcced044/packages/core/src/envelope.ts#L24

Reply:
so in this case the `event.sdk` is already set and when you read the `sdk` in the native layers, you could read directly from `event.sdk` and not from `event. _metadata`, right?
I'd say we could even delete `event. _metadata` from the event before writing it to the disk, it helps with serialization/deserialization/loading the file to memory performance.

Reply:
`event._metadata` doesn't exist, there is `event.sdk` from the JS, yes.
`client_options._metadata` exists and that is what we pass down to initialize the native SDKs with the same client name and version.

Reply:
Indeed, I was somehow comparing the `event` and the `options`, of course, we use the `options` to init the SDK, my bad.